client/webserver/site/src/js/markets.js

```diff
@@ -47,6 +47,8 @@ export default class MarketsPage extends BasePage {
       'baseWalletIcons', 'quoteWalletIcons', 'lotSize', 'rateStep', 'lotField',
       'tifNow', 'mktBuyBox', 'mktBuyLots', 'mktBuyField', 'minMktBuy', 'qtyBox',
       'loaderMsg', 'balanceTable', 'orderPreview',
+      // Balance
+      'marketBalance',
```

Comment:
No new section here. It's the same section as the `balanceTable`, which is the section labeled `// Order form`.